views/utilities.js

```diff
@@ -340,8 +340,13 @@ module.exports  = {
         } 
 
         if (address['county']) {
-            requestData.countyFromZip = address['county'].replace(" County", "");
+            requestData.countyFromZipDropCounty = address['county'].replace(" County", "");
```

Comment:
Why did you insert this new stepping-stone variable?

Reply:
It's not strictly necessary, but I thought it made the code more readable.  Both the with- and without-spaces version of the county name are descended from this var, which is the name without the string " County."